cmd/agent/main.go

```diff
@@ -75,6 +76,57 @@ func main() {
 		"working directory to use when spawned via Persistent Worker")
 	flag.Parse()
 
+	// Initialize Sentry
+	err := sentry.Init(sentry.ClientOptions{
```

Comment:
Do we need to set release similar to how softnet and tart does?

Reply:
See https://github.com/cirruslabs/cirrus-ci-agent/pull/264/commits/e1a0fa1040ca00465b2c25d1e72afcf4743a29f4.